newsroom/templates/scripts.html

```diff
@@ -35,35 +35,36 @@
 <!-- Global site tag (gtag.js) - Google Analytics -->
 <script async src="https://www.googletagmanager.com/gtag/js?id={{ get_setting('google_analytics') | safe }}"></script>
 <script>
-  window.dataLayer = window.dataLayer || [];
-  function gtag(){dataLayer.push(arguments);}
-  gtag('js', new Date());
+    window.dataLayer = window.dataLayer || [];
+    function gtag(){dataLayer.push(arguments);}
+    gtag('js', new Date());
 
-  gtag('set', 'user_properties', {
-      company: (window.profileData || {}).companyName || 'none',
-      user: ((window.profileData || {}).user || {}).first_name || 'unknown',
-  });
+    gtag('set', 'user_properties', {
+        company: (window.profileData || {}).companyName || 'none',
+        user: ((window.profileData || {}).user || {}).first_name || 'unknown',
+    });
 
-  {% if session.get('user') %}
-  gtag('set', {'user_id': '{{ hash(session['user']) | tojson }}' });
-  {% endif %}
+    let configParams = {};
+    {% if session.get('user') %}
+        gtag('set', {'user_id': '{{ hash(session['user']) | tojson }}' });
```

Comment:
I think you have to remove that `hash` here otherwise it will send hash to analytics but that is not what you will see in the UI

Reply:
I left it with the `hash` because it was already there. The one I added is this one actually https://github.com/superdesk/newsroom-core/pull/948/files#diff-2dfa5523a06deb5a874417e434f6bf221d060a1314ce91dc54c3a03a5b4e1837R51 and the one with the hash was this one https://github.com/superdesk/newsroom-core/pull/948/files#diff-2dfa5523a06deb5a874417e434f6bf221d060a1314ce91dc54c3a03a5b4e1837L47-L49

Do you think we should remove the hash? Wouldn't it break anything?

Reply:
you can just set it here without hash and remove the other one, according to docs this is the way to do it
https://developers.google.com/analytics/devguides/collection/ga4/user-id?client_type=gtag#send_user_ids

Reply:
yeah, but my point was that I don't know why the `gtag('set', {'user_id'` was there in the first place with a hashed id. But if you think it is safe to remove it, I'll do it right away.

Reply:
done

Reply:
it was there to obfuscate the real id but provide consistent value for a particular user, but now we want the actual ids